incubator/cassandra/templates/cassandra-statefulset.yaml

```diff
@@ -103,13 +103,19 @@ spec:
         livenessProbe:
           exec:
             command: [ "/bin/sh", "-c", "nodetool status" ]
-          initialDelaySeconds: 90
-          periodSeconds: 30
+          initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds | default 90 }}
```

Comment:
are the defaults needed as long as the values are defined in the values.yaml? they act as defaults?

Reply:
Ah, totally missed what you meant. Still learning Helm, should be fixed now.